scheduler/src/cook/scheduler/scheduler.clj

```diff
@@ -871,10 +871,11 @@
             (ratelimit/spend! quota/per-user-per-pool-launch-rate-limiter token-key 1)
             (ratelimit/spend! compute-cluster-launch-rate-limiter ratelimit/compute-cluster-launch-rate-limiter-key 1))
           (prom/inc prom/scheduler-jobs-launched {:pool pool-name :compute-cluster (cc/compute-cluster-name compute-cluster)})
-          (locking fenzo
-            (-> fenzo
-                (.getTaskAssigner)
-                (.call task-request hostname)))))
+          (when fenzo
```

Comment:
Can you add a comment to the function above saying that `fenzo` can be null (and when).

Reply:
Done